src/Commands/Backup/Automatic/DisableCommand.php

```diff
@@ -15,16 +15,16 @@ class DisableCommand extends TerminusCommand implements SiteAwareInterface
     use SiteAwareTrait;
 
     /**
-     * Disable a regular backup schedule
+     * Disables automatic backups.
      *
      * @authorize
      *
      * @command backup:automatic:disable
      *
-     * @param string $site_env Site & environment to disable the schedule of, in the format `site-name.env`.
+     * @param string $site_env Site & environment in the format `site-name.env`
      *
      * @usage terminus backup:automatic:disable <site>.<env>
-     *    Disables the regular backup schedule for the <env> environment of <site>.
+     *    Disables the regular backup schedule for the <site>'s <env> environment.
```

Comment:
The second "the" should be axed. `<site>` and `<env>` should be interchangeable for any valid site and environment name.
`Disables the regular backup schedule for the bcb-cs61a's dev environment.`